hips/tiles/surveys.py

```diff
@@ -127,7 +132,12 @@ def tile_format(self) -> str:
     @property
     def base_url(self) -> str:
         """HiPS access url"""
-        return self.data['moc_access_url'].rsplit('/', 1)[0]
+        try:
+            return self.data['hips_service_url']
+        except IndexError:
+            return self.data['moc_access_url'].rsplit('/', 1)[0]
+        except:
```

Comment:
2 comments:

- I think this should be nested one level deeper
- what should we do in case we reach the last fallback and still don't get the base URL. Return None, raise an exception? @cdeil , what is the most Pythonic/Astropynic way of handling this?